### PR TITLE
Provide information about how to force install hex and rebar.

### PR DESCRIPTION
**Pull request by myronmarston**
As mentioned in this mailing list thread:

https://groups.google.com/d/msg/elixir-lang-core/JptOMxrLxcs/95mR6YYJDAAJ

...I could not figure out what the command to force install rebar was.
Including the information in the prompt will help users figure this out
without having to ask like I did.

### CODE REVIEW

**Comment by myronmarston** [comment]
The CI failures don't seem related to my changes.

File: lib/mix/lib/mix/hex.ex [hunk 1]
```diff
@@ -14,7 +14,7 @@ defmodule Mix.Hex do
       shell = Mix.shell
       shell.info "Could not find Hex, which is needed to build dependency #{inspect app}"
 
-      if shell.yes?("Shall I install Hex?") do
+      if shell.yes?("Shall I install Hex? (use `mix local.hex --force` if running non-interactively)") do
```
Comment (lexmag) [inline lib/mix/lib/mix/hex.ex]: We don't use backticks in shell messages.

Comment (eksperimental) [inline lib/mix/lib/mix/hex.ex]: or something like this:
> Shall I install Hex? (if running non-interactively use: mix local.hex --force)

**Comment by myronmarston** [comment]
Updated.

> We don't use backticks in shell messages, please use double quotes.

For my understanding, can you explain why you don't use backticks?  I thought it was pretty standard...

**Comment by josevalim** [comment]
> For my understanding, can you explain why you don't use backticks? I
thought it was pretty standard...

At this point we just don't. So while maybe we should, that's another
discussion that is not part of this PR. :)



-- 


*José Valim*
www.plataformatec.com.br
Skype: jv.ptec
Founder and Director of R&D

**Comment by lexmag** [comment]
> For my understanding, can you explain why you don't use backticks? I thought it was pretty standard...

It's simply a consistency matter, and also, there is no markdown processing in the shell. :bowtie:

**Comment by lexmag** [comment]
Thank you. 💛

**Pull request merged**